skills/cairo-auditor/scripts/quality/audit_local_repo.py

```
@@ -309,6 +318,26 @@ def add(rel: str, line: int, class_id: str, severity: str, title: str) -> None:
                     "Class hash mutation lacks explicit non-zero validation",
                 )
 
+        for class_id, detector in benchmark_detectors.items():
+            if not callable(detector):
+                continue
+            if any(row["file"] == rel and row["class_id"] == class_id for row in findings):
+                continue
+            try:
+                detected = bool(detector(code))
+            except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)